### DISCUSSION TITLE
How to search for the visible text of an hyperlink, rather than its actual content?

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1212] [Q&A]
Often web pages show the title of an article, which is hyperlinked to its pdf file or another webpage that contains the article as text, or which contains a link to the pdf file.

If I copy that link, it actually places a lot of content on the clipboard.
When this complex structure is passed to FL, it fails to understand what is the search string.
As a result, there is no search result (the search fails).

How do we address this?

Here is an example, where the human-readable text says
**Cognitive architecture. Designing for how we respond to the built environment**

But if I copy this text, the following content is placed on clipboard:

```
<html>
<body>
<!--StartFragment-->

Cognitive architecture. Designing for how we respond to the built environment | Cognitive architecture. Designing for how we respond to the built environment
-- | --
Cognitive architecture. Designing for how we respond to the built environment

<!--EndFragment-->
</body>
</html>[Cognitive architecture. Designing for how we respond to the built environment](https://booksc.org/book/64395739/6b2b18)
```

**Comment by taooceros** [comment]
We can take a try to override the paste option. But I suspect whether that may work under this situation since the content is already plain text.
